#!/usr/bin/python3

def raise_exception_msg(messgage=""):
    raise NameError(message)
